alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -108,6 +108,17 @@ private function formatBundleContent($bundleContent)
 		{
 			$bundleContent = $this->getIfarmEmbedCode($bundleContent);
 		}
+		else
+		{
+			$config["uiConf"] = $this->playerConfig;
+			$config = json_encode($config);	
+
+			$kalturaPlayerConfig = "
+			var kalturaPlayerConfig = $config;
```

Comment:
how do you expect the player to get the default config? we will need to add some relevant code in player that takes this into consideration when it's loaded.
Currently this kalturaPlayerConfig is just lying there and the app needs to refer to it when calling `KalturaPlayer.setup(..., config);`

Reply:
Also, need to make this config take into consideration a situation where multiple players are on page and we don't want them to run over another